packages/@tailwindcss-vite/src/index.ts

```diff
@@ -239,106 +266,57 @@ export default function tailwindcss(): Plugin[] {
           ...(config.command === 'build' ? ['vite:css-post'] : []),
         ]
 
+        console.log('plugins')
+        console.log(config.plugins)
         cssPlugins = config.plugins.filter((plugin) => {
           return allowedPlugins.includes(plugin.name)
         })
```

Comment:
This is something I don't fully understand yet and needs more thought. Maybe this is no longer necessary since we now run in `pre` and downstream plugins would _just work_?